client/state/action-types.js

```diff
@@ -10,6 +10,9 @@
  *  - https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#actions
  */
 
+export const ANALYTICS_BUMP_STAT = 'ANALYTICS_BUMP_STAT';
+export const ANALYTICS_RECORD_EVENT = 'ANALYTICS_RECORD_EVENT';
+export const ANALYTICS_RECORD_PAGE_VIEW = 'ANALYTICS_RECORD_PAGE_VIEW';
```

Comment:
In this case @gwwar is this really applicable? To me it reads funny and I don't anticipate the same kind of fanning out that other events might have, such as `FETCH/SET/SAVE/ETC_POSTS`

In other words, since the name is directly tied to the action, I'm not sure we'd ever have `ANALYTICS_STAT_RECORD`